packages/webpack/mixins/config/mixin.core.js

```diff
@@ -32,9 +33,19 @@ class WebpackConfigMixin extends Mixin {
     })();
 
     this.configureBuild(webpackConfig, loaderConfigs, target);
-    debugConfig(target, webpackConfig);
 
-    return webpackConfig;
+    const isBuild =
+      target === 'build' ||
+      (target === 'node' && process.env.NODE_ENV === 'production');
```

Comment:
what situation does this line describe?

Reply:
Its about deciding what is a build we are targeting with the `SpeedMeasurePlugin`.
Below you can see that we negate the result so what we are heading for is a develop build and not production.
E.g. yarn start.
But it also should run for develop build of the 'node' target to get the metrics for the server bundle as well.